Gopkg.toml

```diff
@@ -0,0 +1,26 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/russross/blackfriday"
+  version = "1.4"
```

Comment:
If you haven't been following dep closely, this line may surprised you.  In other package managers, this would look like I'd pinned you specifically to v1.4 and yet somehow, the lock file now shows v1.5.  In dep, all constraints are implicitly [carot requirements](http://doc.crates.io/specifying-dependencies.html#caret-requirements).  So this line really means `1.4 <= version < 2.0.0` which is essentially what we want.  `dep ensure -update` respects the existing manifest and only updates the lockfile and vendor folder.